File: frontend/src/types/alert.ts
export type AlertStatus = "NEW" | "ACKNOWLEDGED" | "RESOLVED";

export interface Alert {
  id: string;
  status: AlertStatus;
  created_at: string;
  acknowledged_at?: string | null;
  resolved_at?: string | null;
  sensor_id: string;
  store_id: string;
  resolved_by_user_id?: string | null;
  sensor: {
    name: string;
    zone: {
      name: string;
    };
  };
  resolvedByUser?: {
    first_name: string;
    last_name: string;
  } | null;
}

File: mobile/app/src/main/java/com/auratherm/mobile/model/Alert.kt
package com.auratherm.mobile.model

data class Alert(
    val id: String,
    val status: String,          // "NEW", "ACKNOWLEDGED", "RESOLVED"
    val created_at: String,      // залишаємо назву як у БД і твоєму UI
    val sensor_id: String,
    val store_id: String,
    val sensor: Sensor           // Використовуємо повну модель датчика
)